Adding pdf posts

DevinSmithWork opened the pull request.

myucekul commented on submit_pubs_v2.py:
```diff
@@ -24,19 +26,52 @@ def submit_pubs(new_osti_pubs, osti_creds, submission_limit):
+                                   params=params,
+                                   files={'file': pdf_response.content})
+
+    if media_response.status_code > 300:
```
We should save the response code in DB in case of failure too.

myucekul commented on submit_pubs_v2.py:
```diff
@@ -24,19 +26,52 @@ def submit_pubs(new_osti_pubs, osti_creds, submission_limit):
 
+            # If metadata submission was successful, submit the PDF
+            media_response = submit_media(osti_creds, osti_pub)
+            if media_response is False:
```
How would the failed media submissions be retried?

DevinSmithWork replied:
We have a few options -- can discuss at 1:1

DevinSmithWork commented:
@myucekul -- Code updates from this week have been pulled into this adding-pdf-posts branch:
• Added ability to resubmit metadata
• Added ability to resubmit PDFs
• Significant refactoring
• Sequestered the V1 functions wherever possible to make them easier to remove from the code when E-Link 2 goes live.
